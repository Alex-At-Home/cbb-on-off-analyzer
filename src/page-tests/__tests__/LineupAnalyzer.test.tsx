import React from 'react';
import LineupAnalyzerPage from '../../pages/LineupAnalyzer';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe("LineupAnalyzerPage", () => {
  test("LineupAnalyzerPage - should create snapshot", () => {
    const wrapper = shallow(<LineupAnalyzerPage/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
