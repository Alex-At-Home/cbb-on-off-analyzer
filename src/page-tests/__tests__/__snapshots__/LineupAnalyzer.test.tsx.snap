// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineupAnalyzerPage LineupAnalyzerPage - should create snapshot 1`] = `
<Container
  fluid={false}
>
  <ForwardRef
    noGutters={false}
  >
    <Col
      className="text-center"
      xs={12}
    >
      <h3>
        CBB Lineup Analysis Tool 
        <span
          className="badge badge-pill badge-info"
        >
          BETA!
        </span>
      </h3>
    </Col>
  </ForwardRef>
  <ForwardRef
    noGutters={false}
  >
    <HeaderBar
      common={Object {}}
      thisPage="lineup-"
    />
  </ForwardRef>
  <ForwardRef
    noGutters={false}
  >
    <GenericCollapsibleCard
      summary="Lineups: 2019/20  (M): query:'' (max:50, min-poss:5, sort:desc:off_poss, filter:'')"
      title="Team and Game Filter"
    >
      <LineupFilter
        onChangeState={[Function]}
        onStats={[Function]}
        startingState={Object {}}
      />
    </GenericCollapsibleCard>
  </ForwardRef>
  <ForwardRef
    noGutters={false}
  >
    <GenericCollapsibleCard
      title="Lineup Analysis"
    >
      <LineupStatsTable
        lineupStats={Object {}}
        onChangeState={[Function]}
        rosterStats={
          Object {
            "baseline": Array [],
            "off": Array [],
            "on": Array [],
          }
        }
        startingState={Object {}}
        teamStats={Object {}}
      />
    </GenericCollapsibleCard>
  </ForwardRef>
  <Footer
    server="server"
  />
</Container>
`;
